.github/workflows/release.yml

```diff
@@ -39,8 +39,8 @@ jobs:
     strategy:
       matrix:
         os: ['macos-10.15', 'windows-latest', 'ubuntu-18.04']
-        py-version: ['3.7', '3.8', '3.9']
-        tf-version: ['2.5.1', '2.7.0']
+        py-version: ['3.7', '3.8', '3.9', '3.10']
+        tf-version: ['2.6.2', '2.8.0']
```

Comment:
TensorFlow 2.6 doesn't support Python 3.10, so I think we need to ignore this combination from the build matrix.